tests/testthat/test_access.R

```diff
@@ -1,6 +1,6 @@
 context("Access rules")
 
-mn <- env_load()$mn
+mn <- tryCatch(env_load()$mn, error = function(e) env_load()$mn)
```

Comment:
This is just a stop-gap due to the weird SSL error thing, right? Fine to leave in.

Reply:
yeah I think so - Dominic added that in